examples/00-systemcoupling/turek_hron_fsi2.py

```diff
@@ -0,0 +1,447 @@
+# SOFTWARE.
+
+""".. _ref_turek_hron_fsi2_example:
+
+Turek-Hron FSI2 Benchmark Example
+---------------------------------
+
+This example is a version of the *Turek-Hron FSI2* case that is
+often used as a benchmark case for System Coupling. This two-way, fluid-structure
+interaction (FSI) case is based on co-simulation of a transient oscillating
+beam with surface data transfers.
+
+- Ansys Mechanical APDL (MAPDL) is used to perform a transient structural analysis.
+- Ansys Fluent is used to perform a transient fluid-flow analysis.
+- System Coupling coordinates the coupled solution involving the above products to
+  solve the multiphysics problem via co-simulation.
+
+**Problem description**
+
+An elastic beam structure is attached to a rigid cylinder. The system
+resides within a fluid filled channel:
+
+.. image:: /_static/turek_hron_case.png
+   :width: 400pt
+   :align: center
+
+The flow is laminar with a Reynolds number of :math:`Re = 100`. The inlet velocity
+has a parabolic profile with a maximum value of :math:`1.5\bar{U}`, where :math:'\bar{U}'
+is the average inlet velocity. The cylinder sits at an offset of :math:`0.05~m` to the
+incoming flow, causing an imbalance of surface forces on the elastic beam.
+The beam and the surrounding fluid are simulated for a few time setps to
+allow an examination of the motion of the beam as it starts vibrating due to
+vortices shedded by the rigid cylinder.
+"""
+# %%
+# Import modules, download files, launch products
+# -----------------------------------------------
+# Setting up this example consists of performing imports, downloading
+# the input file, and launching the required products.
+#
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Import ``ansys-systemcoupling-core``, ``ansys-fluent-core`` and
+# ``ansys-mapdl-core`` and other required packages.
+
+# sphinx_gallery_thumbnail_path = '_static/turek_hron_velocity.jpeg'
+
+
+import ansys.fluent.core as pyfluent
+import ansys.mapdl.core as pymapdl
+import matplotlib.pyplot as plt
+
+import ansys.systemcoupling.core as pysyc
+from ansys.systemcoupling.core import examples
+
+# %%
+#
+# Download the input file
+# ~~~~~~~~~~~~~~~~~~~~~~~
+# This example uses one pre-created file - a Fluent mesh file that contains
+# the fluids mesh and named zones.
+#
+fluent_msh_file = examples.download_file(
+    "turek_hron_fluid.msh", "pysystem-coupling/turek-horn-benchmark"
+)
+
+mapdl_cdb_file = examples.download_file(
+    "turek_hron_solid_model.cdb", "pysystem-coupling/turek-horn-benchmark"
+)
+
+
+# %%
+# Launch products
+# ~~~~~~~~~~~~~~~
+# Launch instances of the Mechanical APDL, Fluent, and System Coupling
+# and return *client* (session) objects that allow you to interact with
+# these products via APIs exposed into the current Python environment.
+
+mapdl = pymapdl.launch_mapdl(version="24.2")
+fluent = pyfluent.launch_fluent(product_version="24.2")
+
+syc = pysyc.launch()
```

Comment:
`pysyc.launch(start_output = True)`

and then remove this line below:

`syc.start_output()`

(just more concise that way)